dv/uvm/core_ibex/Makefile

```diff
@@ -480,9 +492,9 @@ $(rtl-sim-logs): \
 	@echo Running RTL simulation at $@
 	$(verb)mkdir -p $(@D)
 	$(verb)env \
-	  SPIKE_CFLAGS="$(SPIKE_CFLAGS)" \
-	  SPIKE_LDFLAGS="$(SPIKE_LDFLAGS)" \
-	  SPIKE_LIBS="$(SPIKE_LIBS)" \
+	  SPIKE_CFLAGS="$(SPIKE_CFLAGS_I) $(SPIKE_CFLAGS_OTHER)" \
+	  SPIKE_LIBS="$(SPIKE_LIBS_lowerL) $(SPIKE_LIBS_upperL)" \
+	  SPIKE_LDFLAGS="$(SPIKE_LIBS_FMT)" \
```

Comment:
I realise this isn't new to this commit, but I don't *think* we should need to pass these flags when running (as opposed to compiling) the testbench? Similarly for `gen_cov` below.

Reply:
Good point! They're only needed for compiling/building the tb. Removed in other steps.